app/controllers/user/attendances_controller.rb

```diff
@@ -43,6 +43,24 @@ def update_zoom_alias
     redirect_to attendance_path(attendance)
   end
 
+  def destroy
+    attendance = Attendance.find(params[:id])
+    module_id = attendance.turing_module.id  
+    attendance_details = {
+      id: attendance.id,
+      turing_module_id: attendance.turing_module_id,
+      user_id: attendance.user_id,
+      meeting_type: attendance.meeting_type,
+      meeting_id: attendance.meeting_id,
+      end_time: attendance.end_time
+    }
+    if attendance.destroy
+      logger.info("Attendance #{attendance.id} deleted by user #{current_user.email}. Deleted Attendance Details: #{attendance_details.inspect}")
+    end
```

Comment:
I like the logging addition. Let's clean this up by using the Active Model `.attributes` method rather than manually creating the details hash.